docs/crocks/Maybe.md

```diff
@@ -0,0 +1,97 @@
+# Maybe
+
+`Maybe c a`
+
+--
+
+--
+
+```js
+--
+```
+
+`Maybe` exposes the following constructor and instances:
+
+| Constructor | Instance |
+|:---|:---|
+| [`Nothing`](#nothing), [`Just`](#just), [`of`](#of) | [`inspect`](#inspect), [`maybe`](#maybe), [`either`](#either), [`option`](#option), [`type`](#type), [`equals`](#equals), [`coalesce`](#coalesce), [`map`](#map), [`ap`](#ap), [`of`](#of), [`chain`](#chain), [`sequence`](#sequence), [`traverse`](#traverse) |
+
+## Constructors
+
+### Nothing
+
+`Maybe m => c -> m c _`
+
+### Just
+
+`Maybe m => a -> m _ a`
+
+### of
+
+`Maybe m => a -> m _ a`
+
+## Instances
+
+### inspect
+
+`() => String`
+
+### maybe
+
+`Maybe m => m c a ~> a -> m a -> a`
+
+### either
+
+`Maybe m => m c a ~> (c -> b) -> (a -> b) -> b`
+
+### option
+
+`Maybe m => m c a ~> a -> m a -> a`
+
+### value
+
+`Maybe m => m c a ~> () => c | a`
+
+### type
+
+`() -> String`
+
+### swap
+
+`Maybe m => m c a ~> m a c`
+
+### coalesce
+
+`Maybe m => m c a ~> (c -> b) -> (a -> b) -> m _ b`
```

Comment:
The first parameter is fixed to (), so this should be:

`Maybe m => m a ~> (() -> b) -> (a -> b) -> m b`